vote_text_drafts/clarify_RPIP-4.md

```diff
@@ -1,32 +1,48 @@
 
+Note: There is no practical way to carry out a vote on initial definition only. That means this vote will determine an _outcome_, so please take that under consideration when selecting your preference. 
+
 ### Outcomes
-- If both this vote and the concurrent `Clarify Voting Criteria (RPIP-4) [10-150% strategy]` votes have matching results, the RPIP Editors are instructed to clarify RPIP-4 per those results
-- If the votes do not match, or quorum fails on either vote, no action is taken and other votes will remain blocked until a future successful attempt to clarify RPIP-4
+- If both concurrent votes (`Clarify Voting Criteria (RPIP-4) [eligible voters: all nodes]` and `Clarify Voting Criteria (RPIP-4) [eligible voters: nodes w/RPL ≥10% borrowed ETH]`) have matching results, the RPIP Editors are instructed to clarify RPIP-4 and RPIP-8 per those results and the `Implementation` section below
+- If the votes do not match, or quorum fails on either vote, no action is taken
+  - Other votes will remain blocked until a future successful attempt to clarify the voting body
 
 ### Implementation
 
-#### Clarify "Implementation" text
+#### Clarify "Implementation" text in RPIP-4
+The main clarification, which specifies the set of nodes eligible to receive vote power.
 - Current text: `To be eligible for vote participation, the RPL must be effectively staked in the Rocket Pool protocol as reported by getNodeEffectiveRPLStake().`
-- Clarify to 0-150%: `RPL staked on a node contributes to vote power up to 150% of the value of the node's bonded ETH.`
-- Clarify to 10-150%: `A node is eligible to get vote power if the RPL staked on it is ≥10% of the value of the node's borrowed ETH. For nodes eligible to receive vote power, RPL staked on a node contributes to vote power up to 150% of the value of the node's bonded ETH.`
+- Clarify to all nodes eligible: `All nodes are eligible to receive vote power. For nodes eligible to receive vote power, RPL staked on a node contributes to vote power up to 150% of the value of the node's bonded ETH.`
+- Clarify to RPL>10% borrowed ETH eligible: `A node is eligible to receive vote power if the RPL staked on it is ≥10% of the value of the node's borrowed ETH. For nodes eligible to receive vote power, RPL staked on a node contributes to vote power up to 150% of the value of the node's bonded ETH.`
 
-#### Clarify "Rationale" text
+#### Clarify "Rationale" text in RPIP-4
+Avoid text that could be read as defining things in the Rationale section.
 - Current text: `RPL is required to be effectively staked for voting to prevent vote-buying by participants unaffiliated with the protocol’s operations.`
 - Clarification: `The RPL eligibility requirements help prevent vote-buying by participants unaffiliated with the protocol’s operations.`
 
+#### Clarify "Specification" text in RPIP-8
```

Comment:
RPIP-8 should probably just be modified to say: "Effectively staked RPL SHALL be used when calculating RPL rewards." It doesn't need to express an opinion on voting power, does it? RPIP-4 already covers the calculation of voting power.

Reply:
Like the single source of truth for voting power should just be RPIP-4? If another RPIP needs to mention it, it should just refer to the calculation in RPIP-4.